src/main/java/com/lalabrand/ecommerce/auth/RefreshToken.java

```diff
@@ -0,0 +1,32 @@
+package com.lalabrand.ecommerce.auth;
+
+import com.lalabrand.ecommerce.user.User;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Table(name = "refresh_token")
+public class RefreshToken {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
```

Comment:
@denomelchenko you don't need a separate `id` and `token` column. Token is a unique UUID value (in your case) that can be a private key.